### ISSUE TITLE
can a result model be fixed if i always ues the same seed?

### ISSUE THREAD

**Issue by FlandrinorZXY** [automl/auto-sklearn#725]
code like this:
clf1 = autosklearn.regression.AutoSklearnRegressor(
                            time_left_for_this_task=30,
                            per_run_time_limit=30,
                            include_estimators=['xgradient_boosting'],
                            include_preprocessors=['no_preprocessing'],
                            ensemble_size=0,
                            delete_tmp_folder_after_terminate=False,
                            seed=1234
                            )
clf1.fit(X0, Y0)
clf1.fit_ensemble(Y_train,ensemble_size=1,ensemble_nbest=1)
y_pred1 = clf1.predict(X_test)
#######################################
i tried many times to see if the same seed=1234 could fix the model trained out...
the result is that though the final preds from different result model were highly similar with each other, these model were not the same one model at all...
so how can i make my train-process reproducible?

**Comment by mfeurer** [comment]
This is possible with the snippet [in this comment](https://github.com/automl/auto-sklearn/issues/514#issuecomment-407781089).
